bin/install.sh

```diff
@@ -17,11 +17,27 @@
 cd  "$( dirname "${BASH_SOURCE[0]}" )/.."
 export IBASE="$(pwd)"
 export WAIT_TIMEOUT=${WAIT_TIMEOUT:-5m}
+ISTIO_CLUSTER_ISGKE=${ISTIO_CLUSTER_ISGKE:-false}
 
 function step() {
     echo "${METHOD}ing $1"
 }
 
+# utility to check for pod existence
```

Comment:
Please remove this part - should be based on values only. It may mess up with other things - we generate a yaml and apply kustomize and may apply it to a different cluster.

Reply:
Ok... I was thinking that this is going to be the basis for the operator and this is what the operator should do, IMO.